python/drisk_api/interfaces/nx.py

```diff
@@ -0,0 +1,33 @@
+"""Export graphs to networkx."""
+try:
+    import networkx as nx
+except ImportError as e:
+    raise ImportError(
+        f"NetworkX is required for this module. Please install it.\n{e}"
+    )
+
+import json
+from io import BytesIO
+from zipfile import ZipFile
+
+import requests
+
+from drisk_api.graph_client import EdgeException, GraphClient
+
+
+def graph_to_networkx(graph: GraphClient) -> nx.DiGraph:
+    """Convert a graph to a NetworkX graph."""
+    r = requests.get(
```

Comment:
Not necessarily for now, but since we probably will keep adding these get requests a lot might be worth adding a custom util `get_request` or something that make the syntax a bit more concise